src/frontend/components/UI/ActionIcons/index.tsx

```diff
@@ -92,6 +96,22 @@ export default React.memo(function ActionIcons({
             data-tour="library-sort-installed"
           />
         </button>
+        {showAlphabetFilter !== undefined && (
```

Comment:
I don't think we need this `showAlphabetFilter !== undefined &&` check here, we want to always show the button to enable/disable the alphabet filter

I don't think it's even possible to have `undefined` here, I think the type of that context property might be wrong, cause I think we are always passing a value there